src/elm-review.ts

```diff
@@ -0,0 +1,217 @@
+      options: [
+        {
+          name: ["--help", "-h"],
+          description: "Show help for elm-review new-package",
+        },
+      ],
+      args: {
+        name: "rule name",
+        description:
+          "Add a rule file in src/ and a corresponding test file in tests/. Should be formatted just like an Elm module name, such as NoDoingThis or No.Doing.This",
+      },
+    },
+  ],
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for elm-review",
+    },
+    {
+      name: "--rules",
+      description:
+        "Run with a subsection of the rules in the configuration. Specify them by their name, and separate them by commas",
+      args: {
+        name: "<rule1,rule2,...>",
+        description: "Run with a subsection of the rules in the configuration",
+      },
+    },
+    {
+      name: "--watch",
+      description:
+        "Re-run elm-review automatically when your project or configuration changes",
+    },
+    {
+      name: "--watch-code",
+      description: "Re-run elm-review automatically when your project changes",
+    },
+    {
+      name: "--elmjson",
+      description: "Specify the path to the elm.json file of the project",
+      args: {
+        name: "<path-to-elm.json>",
+        template: "filepaths",
+      },
+    },
+    {
+      name: "--config",
+      description: "Use the review configuration in the specified directory",
+      args: {
+        name: "<path-to-review-directory>",
+        template: "folders",
+      },
+    },
+    {
+      name: "--template",
+      description: "Use the review configuration from a GitHub repository",
+      args: {
+        name: "<author>/<repo>[/path-to-the-config-folder][#branch-or-commit]",
+      },
+    },
+    {
+      name: "--compiler",
+      description: "Specify the path to the elm compiler",
+      args: {
+        name: "<path-to-elm>",
+        template: "filepaths",
+      },
+    },
+    {
+      name: ["--version", "-v"],
+      description: "Print the version of the elm-review CLI",
+    },
+    {
+      name: "--debug",
+      description:
+        "Add helpful information to debug your configuration or rules",
+    },
+    {
+      name: "--report",
+      description: "Error reports will be in JSON format",
+      args: {
+        name: "json or ndjson",
+        generators: reportType,
+      },
+    },
+    {
+      name: "--no-details",
+      description:
+        "Hide the details from error reports for a more compact view",
+    },
+    {
+      name: "--ignore-dirs",
+      description:
+        "Ignore the reports of all rules for the specified directories",
+      args: {
+        name: "<dir1,dir2,...>",
+        template: "folders",
+      },
+    },
+    {
+      name: "--ignore-files",
+      description: "Ignore the reports of all rules for the specified files",
+      args: {
+        name: "<file1,file2,...>",
+        template: "filepaths",
+      },
```

Comment:
Should also have `isVariadic: true` enabled.
```suggestion
      args: {
        name: "<file1,file2,...>",
        template: "filepaths",
        isVariadic: true,
      },
```

Reply:
Fixed this with the inclusion of the elm-review 2.7.0 changes